### PR TITLE
fix: cis_k8s 5.2.10, cis_eks 4.2.9

### PR DESCRIPTION
**Pull request by olegsu**
### Summary of your changes
Fix cis k8s 5.2.10 and cis eks 4.2.9

Before and after the fix
image

### Related Issues
- Fixes: https://github.com/elastic/cloudbeat/issues/636

### Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

Input example
```json
{
    "subType": "Pod",
    "name": "pod-84c8b475f-w9ckf",
    "resource": {
        "metadata": {
            "name": "pod-84c8b475f-w9ckf",
            "namespace": "default"
        },
        "apiVersion": "v1",
        "kind": "Pod",
        "spec": {
            "nodeName": "kind-multi-worker",
            "containers": [
                {
                    "name": "alpine",
                    "securityContext": {
                        "capabilities": {
                            "drop": [
                                "ALL"
                            ]
                        }
                    }
                }
            ]
        }
    },
    "type": "k8s_object"
}
```

### CODE REVIEW

File: bundle/compliance/policy/kube_api/minimize_assigned_capabilities.rego [hunk 1]
```diff
@@ -22,3 +23,9 @@ finding := result {
 		{"containers": {json.filter(c, ["name", "securityContext"]) | c := data_adapter.containers.app_containers[_]}},
 	)
 }
+
```
Comment (ofiriro3) [inline bundle/compliance/policy/kube_api/minimize_assigned_capabilities.rego:26]: Amazing work 👍

File: bundle/compliance/policy/kube_api/minimize_assigned_capabilities.rego [hunk 2]
```diff
@@ -3,13 +3,14 @@ package compliance.policy.kube_api.minimize_assigned_capabilities
...
+import future.keywords.every
 
-default rule_evaluation = true
+default rule_evaluation = false
 
-rule_evaluation = false {
-	container := data_adapter.containers.app_containers[_]
-	capabilities := object.get(container.securityContext, "capabilities", [])
-	not assert.array_is_empty(capabilities)
+rule_evaluation {
```
Comment (ofiriro3) [inline bundle/compliance/policy/kube_api/minimize_assigned_capabilities.rego:10]: future 😮

**Review by ofiriro3 — APPROVED** [review]
Well done 🙃

**Comment by ofiriro3** [comment]
@olegsu Can you verify with @gurevichdmitry that we fix the integration test if necessary?

**Comment by olegsu** [comment]
> @olegsu Can you verify with @gurevichdmitry that we fix the integration test if necessary?

Those rules was not tested in the integration

**Pull request merged**